lld/ELF/LinkerScript.cpp

```diff
@@ -1389,7 +1392,8 @@ LinkerScript::AddressState::AddressState() {
 // we also handle rest commands like symbol assignments and ASSERTs.
 // Returns a symbol that has changed its section or value, or nullptr if no
```

Comment:
The comment about the return value is out of date.

Reply:
Thx. Fixed